docs/.vuepress/styles/index.styl

```diff
@@ -181,3 +181,10 @@ a {
 .home .hero .action-button {
   color: var(--color-white) !important;
 }
+
+.page ol {
+  list-style: inherit;
+}
+ol li {
+  margin-bottom: 20px;
```

Comment:
Should list margins be absolute? Please double check how they are usually set / whats the current default